problem_builder/templates/html/instructor_tool.html

```diff
@@ -9,8 +9,8 @@
     <div class="data-export-field-container">
       <div class="data-export-field">
         <label>
-          <span>{% trans "Username:" %}</span>
-          <input type="text" name="username" />
+          <span>{% trans "Usernames:" %}</span>
```

Comment:
Nit: would be better to say "Username[s]:" in order to clarify that one username is fine.